participants/Panfil.md

```diff
@@ -0,0 +1,63 @@
+# <Fildown>
+
+## (0) Project Name and Team Slack Handles
+
+- Project Name : Panfil
+- Slack Handles : jerseyjerry
+
+## (1) Brief Project Description
+
+We want to supply storage for IDC log data. We may have expected lots of  log data since IDC data center must produce huge data everyday. We can afford the good price for storing .
+
+
+## (2) Link to Project UI
+
+Link : http://www.panfil.io/
+Submission ID: 680687157
+
+## (3) What does your application/UI do?
+
+Our application is mainly upload and search based. It will be a Filcoin user admin panel. Users can use it easily to upload and retrieve kinds of files in the future.
+
+## (4) If your project is using a curated dataset, which one are you using?
+
+No using curated dataset.
+
+## (5) If your project is not using a curated dataset, please tell us a little bit more about your data by answering the questions below.
+
+We provide the data from our IDC switch log Now. 
```

Comment:
Can you explain why the IDC switch log data is useful for any users?  What can they do with that data?

Reply:
Actually, I have no idea about the value of the  IDC switch log. However, the IDC data center boss want to use the Filecoin store service

Reply:
To be eligible for the competition, it needs to be useful to others in some way.  Please check and let us know!

Reply:
hi, I asked  the IDC boss, they reply to me about the use just now:  they need huge data store for the IDC switch data store for some reason here: the log review, safety analyse, active defense strategy. The idc switch logs are kind of cold data.  Others can also use the huge data for big data analyse.
I just think: I use the filecoin to supply the place to store. The IDC datacenter need place to store their data. Thats what the market  is. Every customer have their own reason. Seems github cannot upload picture. I will post the talk between the idc boos and me at the slack. https://files.slack.com/files-pri/TEHTVS1L6-F01CXDM9W4T/idc.jpeg